lib/frontend/src/Obelisk/Frontend.hs

```diff
@@ -0,0 +1,16 @@
+{-# LANGUAGE DeriveFunctor #-}
+{-# LANGUAGE DeriveFoldable #-}
+{-# LANGUAGE DeriveTraversable #-}
+{-# LANGUAGE Rank2Types #-}
+
+module Obelisk.Frontend where
+
+import Data.ByteString (ByteString)
+import Reflex.Dom
+
+-- | Specify a reflex frontend.
+data ObeliskFrontend = ObeliskFrontend
+  { _obeliskFrontend_body :: forall x. Widget x ()
+  -- ^ The content of <body> after the JavaScript is loaded
```

Comment:
Do we want to support mounting to a dom element as well?

Reply:
I think we should, but I refrained from adding that to this PR

Reply:
Ok sounds good.